integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -32,15 +32,15 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   ALLUXIO_JAVA_OPTS: |-
-    {{if $isSingleMaster}}-Dalluxio.master.hostname={{ $fullName }}-{{ $defaultMasterName }}{{end}} -Dalluxio.master.journal.type={{ .Values.journal.type }} -Dalluxio.master.journal.folder={{ .Values.journal.folder }} {{if $isHaEmbedded}}-Dalluxio.master.embedded.journal.addresses={{range $i := until $masterCount}}{{ $fullName }}-master-{{$i}}:19200,{{end}}{{end}} {{ .Values.jvmOptions }} {{- range $key, $val := .Values.properties }} -D{{ $key }}={{ $val }} {{ end }}
+    -Dalluxio.master.journal.type={{ .Values.journal.type }} -Dalluxio.master.journal.folder={{ .Values.journal.folder }} {{if $isHaEmbedded}}-Dalluxio.master.embedded.journal.addresses={{range $i := until $masterCount}}{{ $fullName }}-master-{{$i}}:19200,{{end}}{{end}} {{ .Values.jvmOptions }} {{- range $key, $val := .Values.properties }} -D{{ $key }}={{ $val }} {{ end }}
   ALLUXIO_MASTER_JAVA_OPTS: |-
-    -Dalluxio.master.web.bind.host=0.0.0.0 {{ .Values.master.jvmOptions }} {{- range $key, $val := .Values.master.properties }} -D{{ $key }}={{ $val }} {{ end }}
+    -Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME} -Dalluxio.master.web.bind.host=0.0.0.0 {{ .Values.master.jvmOptions }} {{- range $key, $val := .Values.master.properties }} -D{{ $key }}={{ $val }} {{ end }}
   ALLUXIO_JOB_MASTER_JAVA_OPTS: |-
-    -Dalluxio.job.master.bind.host=0.0.0.0 {{ .Values.jobMaster.jvmOptions }} {{- range $key, $val := .Values.jobMaster.properties }} -D{{ $key }}={{ $val }} {{ end }}
+    -Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME} -Dalluxio.job.master.bind.host=0.0.0.0 {{ .Values.jobMaster.jvmOptions }} {{- range $key, $val := .Values.jobMaster.properties }} -D{{ $key }}={{ $val }} {{ end }}
```

Comment:
Same approach as ALLUXIO_USER_HOSTNAME, we create the env variable in Pod `env` field and refer to it in configmap. 

This item has been mentioned in the CHANGELOG.